model/src/main/java/com/basistech/rosette/apimodel/Request.java

```diff
@@ -17,17 +17,21 @@
 package com.basistech.rosette.apimodel;
 
 import lombok.AllArgsConstructor;
-import lombok.Data;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+import lombok.experimental.SuperBuilder;
 
 /**
  * Common base class of all requests.
  */
-@Data
+@Value
+@NonFinal
+@SuperBuilder
```

Comment:
I noticed SuperBuilder was added to the Names request side of the hierarchy. The doc says it's not compatible with Builder, which DocRequest uses. What part not compatible do you know? Seems like if we should be using SuperBuilder everywhere.

Reply:
I take back, option objects do use `Builder`, but not `DocRequest` or any other request objects.